examples/android/app/src/main/jni/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.22)
+project(sfml-example)
+
+set(SFML_DIR "${CMAKE_ANDROID_NDK}/sources/third_party/sfml/lib/${CMAKE_ANDROID_ARCH_ABI}/cmake/SFML")
```

Comment:
Is this a standard android path or something specific to your particular machine?

Reply:
It is the default value for CMAKE_INSTALL_PREFIX when building SFML: https://github.com/SFML/SFML/blob/master/CMakeLists.txt#L92

So as long as the user doesn't manually specify another path, it will be where SFML is installed to. It is a standard location for android libraries to install to, so I imagine most people will keep the path.

The same goes for the openal.so path, it is the path when SFML places that file by default.

Reply:
Ah okay I see. I didn't know the android build did that. We can keep this.